compose/snippets/src/main/java/com/example/compose/snippets/components/SwipeToDismissBox.kt

```diff
@@ -0,0 +1,320 @@
+                }
+                SwipeToDismissBoxValue.Settled -> {
+                    false
+                }
+            }
+        }
+    )
+
+    SwipeToDismissBox(
+        modifier = Modifier,
+        state = swipeToDismissState,
+        backgroundContent = {
+            // Cross-fade the background color as the drag gesture progresses.
+            val color by animateColorAsState(
+                when (swipeToDismissState.targetValue) {
+                    SwipeToDismissBoxValue.Settled -> Color.LightGray
+                    SwipeToDismissBoxValue.StartToEnd ->
+                        lerp(Color.LightGray, Color.Blue, swipeToDismissState.progress)
+
+                    SwipeToDismissBoxValue.EndToStart ->
+                        lerp(Color.LightGray, Color.Red, swipeToDismissState.progress)
+                },
+                label = "swipeable card item background color"
+            )
+            Row(
+                modifier = Modifier
+                    .background(color)
+                    .fillMaxSize(),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween
+            ) {
+                when (swipeToDismissState.dismissDirection) {
+                    SwipeToDismissBoxValue.StartToEnd -> {
+                        if (value.isItemDone) {
+                            Icon(
+                                imageVector = Icons.Default.CheckBox,
+                                contentDescription = "Item done",
+                                tint = Color.White,
+                                modifier = Modifier
+                                    .padding(12.dp)
+                            )
+                        } else {
+                            Icon(
+                                imageVector = Icons.Default.CheckBoxOutlineBlank,
+                                contentDescription = "Item not done",
+                                tint = Color.White,
+                                modifier = Modifier
+                                    .padding(12.dp)
+                            )
+                        }
+                    }
+
+                    SwipeToDismissBoxValue.EndToStart -> {
+                        Spacer(modifier = Modifier)
+                        Icon(
+                            imageVector = Icons.Default.Delete,
+                            contentDescription = "Remove item",
+                            tint = Color.White,
+                            modifier = Modifier
+                                .padding(12.dp)
+                        )
+                    }
+
+                    SwipeToDismissBoxValue.Settled -> {}
+                }
+            }
+        }
+    ) {
+        content(value)
+    }
+}
+// [END android_compose_components_swipecarditem]
+
+// [START android_compose_components_swipecarditemexample]
+@Preview
+@Composable
+private fun SwipeCardItemExample() {
```

Comment:
Can you add to the Example list activity top compose components?

Reply:
Done.